configure.py

```diff
@@ -323,6 +323,8 @@ def binary(name):
               # Disable size_t -> int truncation warning.
```

Comment:
Pretty sure configure.py is deprecated. Might be better to just remove it entirely.

Reply:
We're using CMake for RapidJSON dependency management now, so configure.py doesn't matter any more.